banzai/tests/data/configdb_example.json

```diff
@@ -555,7 +555,7 @@
               "instrument_set": [
                 {
                   "id": 66,
-                  "state": "SCHEDULABLE",
+                  "state": "STANDBY",
```

Comment:
This technically does what you want without breaking end to end tests but I'm not a big fan.

Reply:
For the record, David and I had a conversation about this. He thought I was changing all schedulable instruments to standby. That's not the case - just changing kb98. The goal of this is to replicate a realistic ConfigDB configuration.